io-engine/src/bdev/nexus/nexus_bdev_children.rs

```diff
@@ -66,6 +66,11 @@ use spdk_rs::{
     IoDeviceChannelTraverse,
 };
 
+
+use events_api::event::EventAction;
+
+use crate::eventing::{EventMetaGen, EventWithMeta};
```

Comment:
```suggestion
use events_api::event::EventAction;
```

And move the "user crate" up together with the other "use crate"

Reply:
Done